hr_experience/views/hr_employee_view.xml

```diff
@@ -64,7 +65,7 @@
                         </form>
                     </field>
                 </page>
-                <page string="Certifications" groups="base.group_user,base.group_hr_user">
+                <page string="Certifications" groups="base.group_user,hr.group_hr_user">
```

Comment:
As 'hr.group_hr_user' has the base rights from 'base.group_user' it can be removed